backend/server.js

```diff
@@ -365,6 +338,14 @@ async function onSendToMessengerButtonClick(sender, userPageId, b64ref) {
     loginKeys.add(userObject.loginKey);
     existingData.loginKeys = Array.from(loginKeys);
 
+    macros.log('neal young', reqs, userObject.loginKey, reqs[userObject.loginKey]);
+    reqs[userObject.loginKey].send((JSON.stringify({
```

Comment:
What if `userObject.loginKey` doesn't exist on reqs? Want to surround this with a quick check like: 

```js
if (reqs[userObject.loginKey]) {...}
```